### ISSUE TITLE
BUG - Sev-2 -all - analytic to capture screen readers not sufficient

### ISSUE THREAD

**Issue by g-opzt** [department-of-veterans-affairs/va-mobile-app#6742]
## What happened?

Current analytics only capture and log a user_property of being a screen reader user if the screen reader is on before the user opens the app. If the screen reader is turned on after app launch, these users are not being appropriately logged.

## Desired behavior

Determine an analytic that can listen for and track that a user has used a screen reader. 

@Sparowhawk provided this suggested solution on Slack:

> we do have a hook we potentially can use in const screenReaderEnabled = useIsScreenReaderEnabled(true) which has a listener to see if they turn it on or off. Maybe a separate analytic would be more accurate so we can more easily track people who start the app with the screen reader enabled and people who turn it on or off afterwards

## Acceptance Criteria

- [ ] App analytics tracks all screen reader users regardless if they turn it on before or after opening the app.

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: Low 
- Frequency: High 

- 2 - High

## Screen shot(s) and additional information

Here is what Dylan shared about how the current analytic functions:

useEffect(() => {
    // only run on app load
    dispatch(sendUsesLargeTextAnalytics())
    dispatch(sendUsesScreenReaderAnalytics())
  }, [dispatch])
This is in the app.tsx file. So when the app is launched(not from background) it will run this check.

export const sendUsesLargeTextAnalytics = (): AppThunk => async () => {
  const islargeText = PixelRatio.getFontScale() > 1
  await setAnalyticsUserProperty(UserAnalytics.vama_uses_large_text(islargeText))
}
/**
 * Redux action to send to analytics if the users is using screen reader
 */
export const sendUsesScreenReaderAnalytics = (): AppThunk => async () => {
  const isRunning = await RNCheckVoiceOver.isVoiceOverRunning()
  await setAnalyticsUserProperty(UserAnalytics.vama_uses_screen_reader(isRunning))
}

## Ticket Checklist
- [ ] _Steps to reproduce are defined (not applicable)_
- [X] Desired behavior is added
- [X] Labels added (front-end, back-end, relevant feature, accessibility, etc)
- [X] Estimate of 1 added (for front-end tickets)
- [X] Added either to the relevant feature epic (if found during new feature implementation), or the [bug epic](https://app.zenhub.com/workspaces/va-mobile-product-view-610035bc5395bb000e62e529/issues/gh/department-of-veterans-affairs/va-mobile-app/5367)

**Comment by bischoffa** [comment]
Carryover Sprint 81 into 82

**Comment by TKDickson** [comment]
@dumathane is this the analytics ticket you were asking about in Slack? afaik, this can be closed/merged without QA or @g-opzt weighing in.

**Comment by TKDickson** [comment]
@dumathane ping
